package.json

```diff
@@ -3,26 +3,31 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@testing-library/jest-dom": "^5.15.0",
-    "@testing-library/react": "^11.2.7",
-    "@testing-library/user-event": "^12.8.3",
-    "@types/jest": "^26.0.24",
-    "@types/node": "^12.20.37",
-    "@types/react": "^17.0.34",
-    "@types/react-dom": "^17.0.11",
-    "eslint-config-react-app": "^7.0.0",
-    "many": "l-1-labs/many-js",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-scripts": "4.0.3",
-    "typescript": "^4.4.4",
-    "web-vitals": "^1.1.2"
+    "@liftedinit/many-js": "^0.1.0-alpha.5",
+    "@liftedinit/ui": "github:liftedinit/lifted-ui",
+    "buffer": "^6.0.3",
+    "react": "^18.2.0",
+    "react-app-rewired": "^2.2.1",
```

Comment:
React App Rewired allows us to modify Webpack (see `config-overrides.js` above) without ejecting from Create React App.